source/docs/articles/sites/backups/restoring-an-environment-from-a-backup.md

```diff
@@ -36,12 +36,23 @@ If you want to download a backup using wget, put the provided temporary link in
 
     wget "https://pantheon-backups.s3.amazonaws.com..."
 
+### Restoring The Codebase
```

Comment:
@bmackinney - I'd suggest reverting the changes made on this doc to avoid encouraging users to restore environments from their local machines. The existing content recommends the supported one-click method. We can link to the new section (`/docs/articles/sites/backups/backup-creation/#about-your-code-archives`) instead. Thoughts?